cmd/abapEnvironmentRunAUnitTest.go

```diff
@@ -446,34 +547,158 @@ type Duration struct {
 
 //ObjectSet in form of packages and software components to be checked
 type ObjectSet struct {
-	Type string `json:"type,omitempty"`
-	Set  []Set  `json:"set,omitempty"`
+	PackageNames     []AUnitPackage   `json:"packagenames,omitempty"`
+	ComponentNames   []Component      `json:"componentnames,omitempty"`
+	Type             string           `json:"type,omitempty"`
+	MultiPropertySet MultiPropertySet `json:"multipropertyset,omitempty"`
+	Set              []Set            `json:"set,omitempty"`
+}
+
+//MultiPropertySet that can possibly contain any subsets/object of the OSL
+type MultiPropertySet struct {
```

Comment:
I think all the "Sets" can be left out.

Reply:
There is a property called "softwareComponent". We should include it.

Reply:
Other sets have been removed, which means that no subsets inside any given MPS might be detected and printed out to the user for further configuration information.
SoftwareComponents are already included with the ComponentNames property which is also mentioned in the example section in the documentation

Reply:
if the OSL set uses "softwareComponents", the yml file should also use "softwareComponents". This is also the term that the Steampunk customer knows.